cmsplugin_articles_ai/management/commands/publish_test_articles.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
Suggestions. Haven't seen other apps do that so not sure what did you have in mind. Though I understand the possible name clashing (although not that probable in this case IMO).